keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -339,37 +363,44 @@ _Figure: Topology Manager components._
 ![topology-manager-instantiation](https://user-images.githubusercontent.com/379372/47447526-945a7580-d772-11e8-9761-5213d745e852.png)
 
 _Figure: Topology Manager instantiation and inclusion in pod admit lifecycle._
- 
+
 ### Feature Gate and Kubelet Flags
- 
+
 A new feature gate will be added to enable the Topology Manager feature. This feature gate will be enabled in Kubelet, and will be disabled by default in the Alpha release.  
 
  * Proposed Feature Gate:  
   `--feature-gate=TopologyManager=true`  
- 
+
  This will be also followed by a Kubelet Flag for the Topology Manager Policy, which is described above. The `none` policy will be the default policy.
- 
+
  * Proposed Policy Flag:  
- `--topology-manager-policy=none|best-effort|restricted|single-numa-node`  
- 
+ `--topology-manager-policy=none|best-effort|restricted|single-numa-node|dynamic`  
+
+ NOTE: 
+ - The Policy Flag will be deprecated in Beta(v1.20), since the Pod Spec starts to specify topology policy. See detales in [Deprecates the Policy Flag of Topology Manager](#deprecates-the-policy-flag-of-topology-manager)
```

Comment:
Dynamic policy wasn't mentioned earlier and in this paragraph it's also not explained what it is. I know that it is explained later.

I would add one more sentence to the `NOTE:`
` - Until Beta(v1.20) to use a topology policy that is specified in a Pod Spec, **dynamic** policy must be configured by a Kubelet flag.

Reply:
Sure, It would be a greate enhancement of this doc :)
Plese commit.